doc/threat_model/Untrusted_Capsule_Isolation.md

```diff
@@ -0,0 +1,23 @@
+Untrusted Capsule Isolation
+===========================
+
+## Isolation Mechanism
+
+Untrusted capsules are limited to what they should be able to access within
+Rust's type system without using `unsafe`. That isolation is implemented by
+banning `unsafe` from use in untrusted capsule code. This isolation is
+vulnerable to code that exploits compiler bugs or bugs in `unsafe` code in
+libraries. When a board integrator chooses to use an untrusted capsule, they are
```

Comment:
```suggestion
vulnerable to code that exploits compiler bugs.
When a board integrator chooses to use an untrusted capsule, they are
```

Reply:
This isolation is vulnerable to exploits in `libcore`'s `unsafe` sections.